k8s/helm-chart/dynatrace-gcp-function/values.yaml

```diff
@@ -27,11 +27,22 @@ deploymentType: "all"
 dynatraceAccessKey: ""
 
 # LOGS VALUES - REQUIRED for 'logs' and 'all' deployment
-# ActiveGate endpoint used to ingest logs to Dynatrace, for example: https://environemnt-active-gate-url:9999/e/environment-id"
-dynatraceLogIngestUrl: ""
+
 # subscription id of log sink pubsub subscription
 logsSubscriptionId: ""
 
+#dynatraceLogIngestUrl ActiveGate endpoint used to ingest logs to Dynatrace, for example: https://environemnt-active-gate-url:9999/e/environment-id"
+#used ONLY when the activeGate.autoDeploy is false
+dynatraceLogIngestUrl: ""
+
+activeGate:
+  #autoDeploy if set to true the Active Gate will be deployed in k8s. In that case also the activeGate.dynatracePaasToken must be filled.
+  autoDeploy: true
```

Comment:
name and logic of new patrameter is not consistant wit Azure logs forwarder.
In Azure AG it is reverse :
`            "--use-existing-active-gate")
                USE_EXISTING_ACTIVE_GATE=true`
https://github.com/dynatrace-oss/dynatrace-azure-log-forwarder/blob/master/deployment/dynatrace-azure-logs.sh#L107

Reply:
Changed to match other clouds.